cmd/common-main.go

```diff
@@ -223,7 +223,11 @@ func handleCommonEnvVars() {
 	} else {
 		// Add found interfaces IP address to global domain IPS,
 		// loopback addresses will be naturally dropped.
-		updateDomainIPs(mustGetLocalIP4())
+		domainIPs := mustGetLocalIP4()
```

Comment:
Does this mean the host is considered to have it's local IPv4s +  whatever public IPs are passed via envs to MinIO ?

Reply:
currently, we had to resort to PUBLIC_IPS to reference the bucket back to the cluster - if that is not specified assume that local IP + command-line hosts are the bucket mapped entities.